Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -1465,6 +1465,9 @@ void registerAppInit(@NonNull ServerRequestInitSession request, boolean ignoreWa
          initTasks(request, ignoreWaitLocks);
 
          requestQueue_.processNextQueueItem("registerAppInit");
+
+         AttributionReportingManager attributionManager = AttributionReportingManager.INSTANCE;
+         attributionManager.registerTrigger(context_, request);
```

Comment:
Does there need to be a opt in/enabled check before we try this?

Reply:
I believe the checkMeasurementApiStatus in the init is all we need since that checks if the device is on the right version and has the measurement API enabled.